plugins/auth-backend-module-atlassian-provider/src/authenticator.ts

```diff
@@ -29,6 +29,7 @@ export const atlassianAuthenticator = createOAuthAuthenticator({
   initialize({ callbackUrl, config }) {
     const clientId = config.getString('clientId');
     const clientSecret = config.getString('clientSecret');
+    const scope = config.getString('scopes');
```

Comment:
Could this be `getOptionalString` and have a sensible default value? It must we require every  adopter to set this explicitly?

Also must be added to [the config schema](https://github.com/backstage/backstage/blob/34217c5d377991d05f724a4c01c44e33573e7812/plugins/auth-backend-module-atlassian-provider/config.d.ts#L21)

Reply:
Yes good call, done.